src/Search/SearchResults.php

```diff
@@ -182,10 +205,23 @@ private function processResults(): void
             );
             $this->facetInfo['digitalResourceTypeDescriptorsInfo'] = $this->digitalResourceTypeDescriptorRepository->getDigitalResourceTypeDescriptorsInfo($digitalResourceTypeDescriptorBucket);
         }
-
-        $researchGroupBucket = $this->getResearchGroupBucket($aggregations);
+        $researchGroupBucket = $this->combineBuckets($aggregations, 'research_group_aggregation', 'research_groups_aggregation');
+        $fundingOrgBucket = $this->combineBuckets($aggregations, 'funding_organization_aggregation', 'funding_organizations_aggregation');
 
         $this->facetInfo['researchGroupInfo'] = $this->researchGroupRepository->getResearchGroupsInfo($researchGroupBucket);
+        $this->facetInfo['fundingOrgInfo'] = $this->fundingOrganizationRepository->getFundingOrgInfo($fundingOrgBucket);
+    }
+
+    private function getDatasetTypeInfo($dataTypeBucket): array
+    {
+        $dataTypeInfo = [];
+        foreach ($dataTypeBucket as $type => $count) {
+            $typeInfo['id'] = $type;
+            $typeInfo['name'] = $type;
+            $typeInfo['count'] = $count;
+            $dataTypeInfo[] = $typeInfo;
+        }
+        return $dataTypeInfo;
     }
 
     private function getFacetInfo(string $facet, array $facetAgregation): ?array
```

Comment:
Needs docblock

Reply:
This function isn't used anymore, so it's removed.